backend/scripts/imageconversion.py

```diff
@@ -1,65 +1,65 @@
 import os
 import subprocess
 import sys
+from pathlib import Path
 
 from tqdm import tqdm
 
 
-def convert(old_path, new_path, old_filetype, new_filetype, commands=None):
+def convert(src_dir_path, dest_dir_path, commands=['-quality', '20%']):
     """
     Python script for image conversion using Image Magick
     
-    :param old_path: (str) path to directory containing images
-            [ex. 'Users/bob/Desktop/']
-    :param new_path: (str) path where output is intended to be saved in
-            [ex. 'Users/bob/Desktop/new/']
-    :param old_filetype: (str) image format of input images
-            [ex. 'TIF']
-    :param new_filetype: (str) desired output image format
-            [ex. 'jpg']
-    :param commands: (list) commands to apply to images, default = None
-            [ex. ['-quality, '20%']
+    :param src_dir_path: Path object to directory containing subdirectories of images
+    :param dest_dir_path: Path path object where output is intended to be saved in
+    :param commands: (list of strings) commands to apply to images
```

Comment:
By the way @czheng10 -- I got rid of the two params specifying filetype bc we're always dealing with jp2 images.